types/selenium-webdriver/http.d.ts

```diff
@@ -21,6 +21,8 @@ export class Request {
      */
     constructor(method: string, path: string, opt_data?: Object);
 
+    headers: Map<string, string>;
+
```

Comment:
method: string
path: string
data: Object

Reply:
I will add these fields below the headers definition.